packages/attendance/src/pages/sms/MessageHistory/Molecule/Message.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
```

Comment:
I think this component has no functional action so it becomes atom not molecule

Reply:
it is only specific to the MessageHistory component, it is not used anywere else beacuse of that i kept is as a molecule instead of atom